app.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Reaction Commerce",
+  "description": "Reaction is a modern reactive, real-time event driven ecommerce platform.",
+  "repository": "https://github.com/reactioncommerce/reaction",
+  "logo": "https://avatars1.githubusercontent.com/u/5605462?s=200&v=4",
+  "website": "https://www.reactioncommerce.com",
+  "keywords": ["reaction", "e-commerce", "meteor"],
+  "buildpacks": [
+    {
+      "url": "https://github.com/Zanobo/reaction-buildpack.git"
+    }
+  ],
+  "addons": [
+    "papertrail",
+    {
+      "plan": "mongolab:sandbox"
+    }
+  ],
+  "env": {
+    "TOOL_NODE_FLAGS": {
+      "required": true,
+      "description": "Set memory limit to a high value",
```

Comment:
Actually what we are doing here is setting the limit to a lower level so that it does not consume all memory on the machine. Setting it higher will actually cause it to crash. So we might want to change that description so it doesn't confuse people